code/src/components/NewTask.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import uniqid from "uniqid";
+
+import "./NewTaskStyle.css";
+import todos from "../reducers/todos";
+
+const NewTask = () => {
+  const [inputValue, setInputValue] = useState("");
+
+  const dispatch = useDispatch();
+
+  const handleChange = (event) => {
+    setInputValue(event.target.value);
+  };
+
+  let newTodo;
+
+  const handleOnClick = (event) => {
+    event.preventDefault();
+    if (inputValue === "") {
+      alert("You can't add an empty todo.");
```

Comment:
Great that you made so the user can't input an empty task. When the alert has pop-ed up, the page is blank.. so a suggestion could be to take the user back to the app to try again instead of the blank page. :-)